compose/cli/main.py

```diff
@@ -137,7 +137,7 @@ def get_filtered_args(args):
 
 
 def exit_with_metrics(command, log_msg=None, status=Status.SUCCESS, exit_code=1):
-    if log_msg:
+    if log_msg and command != 'exec':
```

Comment:
Perhaps I'm driving by, but, this was a useful error? Now, when you execute commands that rely on docker containers, they exit 0 as if everything is fine. I'm sorry for being lazy, but, why is this an improvement? @ulyssessouza